demo/full/scripts/controllers/ContentList.jsx

```diff
@@ -683,14 +679,14 @@ class ContentList extends React.Component {
             }
           </div>
           <div className="choice-input-button-wrapper">
-            <div className="auto-play">
-              AutoPlay
+          <div className="auto-play">
+              Options
               <label className="input switch">
                 <input
                   type="checkbox"
-                  aria-label="Enable/Disable AutoPlay"
-                  checked={autoPlay}
-                  onChange={onAutoPlayClick}
+                  aria-label="Show/Hide player options"
```

Comment:
I would think that for a checkbox, the aria-label should only talk about what happen when you check it.

I'm not 100% sure, but here someone could be unsure if the checking is for showing or hiding. I would guess that checkboxes are properly anounced by screenreaders (though I never used one so I might be wrong).

Reply:
Yeah, u are right, I added a little conditions to either write the word `Show` or `Hide` depending of the context of the state.

Reply:
Again I'm not sure if the aria-label should not just be `Show player options`. I would have guessed that screen-readers (for which aria-label are generally for) also communicate in some ways the type of input - in which case changing the aria-label may be confusing - though I may be wrong.

Examples online seem to prefer setting the `aria-label` of checkboxes to something like `Select to show options` which also gives the information that the input has to be selected (it doesn't change if the input is un-selected).